css-light/base.js

```diff
@@ -0,0 +1,13 @@
+import {Component} from 'react';
+import {css, inject} from 'css-light';
+
+export class Comp extends Component {
+    componentWillMount() {
```

Comment:
Side-effects in `componentWillMount` are discouraged.
Also, can't this be done once in the constructor?

Reply:
In general, it could be done differently, but in this example the `css = () => ({/* ... */});` property
does not exist yet when the parent constructor is called.